pkg/netpol/connlist/conns_formatter.go

```diff
@@ -124,3 +145,63 @@ func getRepresentativePodString(podLabels map[string]string, txtOutFlag bool) st
 	}
 	return res
 }
+
+// following code is common for md, csv and json:
```

Comment:
why not also for `txt`?

Reply:
changes were made (with the consistency of all formats, these are common now also for txt)